src/zzz_od/api/routers/accounts.py

```
@@ -174,156 +413,732 @@ def create_instance(payload: Dict[str, Any] | None = None):
+
+    for instance_id in request.instance_ids:
+        try:
+            if request.operation == "activate":
+                instance = _get_instance_or_404(instance_id, odc)
+                ctx.switch_instance(instance.idx)
+                results.append({"instance_id": instance_id, "success": True})
+                success_count += 1
+
+            elif request.operation == "deactivate":
+                instance = _get_instance_or_404(instance_id, odc)
+                # 停用实例的逻辑（如果有的话）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)